### PR TITLE
fix: webpack browser hrtime

### PR DESCRIPTION
**Pull request by vltansky**
parse-request crashes in Angular on `process is not defined`
image

I tried to make some tests and I guess it depends on which bundler you are using, as it did work for me in browserify (probably they are emulating `process` variable). But Angular+webpack crashes. Didn't test it in React, but I think it will crash there too.

`browser-process-hrtime` looks like not maintained anymore (last commit was 2 years ago and open pull request from December 2018).
So I created a similar package [browser-hrtime](https://www.npmjs.com/package/browser-hrtime) that solves this problem (also add support for process.hrtime.bigint(), tests for browser, and in general made my best to make it good enough).

So I propose you to take a look, and of course here is a screenshot after fix:
image

### CODE REVIEW

File: test/browser.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const dom = new JSDOM(``, {
   runScripts: 'dangerously',
   virtualConsole
 });
+console.log(dom.window);
```
Comment (niftylettuce) [inline test/browser.js:23]: @vltansky Can you please remove this?

**Review by niftylettuce — APPROVED** [review]

**Comment by niftylettuce** [comment]
Published v4.0.0 to npm.  Thank you @vltansky for your contribution.  If you would like to move your package to @cabinjs organization on GitHub, please submit a transfer request.  We would also love to have contributor access to the package, which you could add us with `npm owner add niftylettuce browser-hrtime`.

Also, Cabin v7.1.0 is released with latest version of `parse-request@4.0.0` to npm.

https://cabinjs.com

**Comment by vltansky** [comment]
Hi, thanks.
Yeah, I'd be happy to move package to your organization. There are a lot of things that I wanted to polish there (especially tests) and would be happy to work on this together :)

Can you guide me how do I transfer repo? According to docs I need to have write permission in organization to transfer it.

image

> Users must have repository creation permissions within the receiving organization before they can transfer a repository that they individually own. If Allow members to create repositories for this organization has been disabled in the receiving organization, only organization owners can transfer repositories into the organization.

I've added you to package collaborators (`npm owner add niftylettuce browser-hrtime`)

File: test/browser.js (hunk 1 above)
Comment (vltansky) [reply test/browser.js:23]: Sorry for this

**Comment by niftylettuce** [comment]
Sent you an invite!

**Pull request merged**